### ISSUE TITLE
/group_report interleaves stoppage and regression alerts

### ISSUE THREAD

**Issue by natorion** [catapult-project/catapult#3471]
E.g. https://chromeperf.appspot.com/group_report?rev=460218

Tests are enumerated but the actual regression size is not shown.

**Comment by eakuefner** [comment]
Probably this got broken in some of the speed releasing churn; I'll investigate and figure out a fix.

**Comment by eakuefner** [comment]
Okay, my initial hunch was wrong, here's an example of a group report that does show deltas: https://chromeperf.appspot.com/group_report?sid=fe1c846cdd8b94571d0c4c97482816f6556c61eb1a8c3097726e2bcdd19c36b0

**Comment by simonhatch** [comment]
Guess it depends on what people like @natorion expect to see on that page?

**Comment by natorion** [comment]
What is a stoppage alert?

**Comment by eakuefner** [comment]
Great question! Stoppage alerts are a kind of non-numerical alert that indicate that data has stopped coming in for a particular test. An implementation detail of the alerts-table element helps define how to display the columns that are unique to both kinds of alerts (StoppageAlert and Anomaly), but the delta columns are shown only if all alerts are Anomaly instances.

In essence, if you are not already concerned about them you probably don't need to start being concerned right now.

Broadly: since @natorion wrote his comment, though, we came up with I think what's probably the best possible fix here: we should display stoppage alerts not if a revision is passed, but if other parameters are passed. And in the case where we have interleaved alerts as reported, we can give preferential treatment to `Anomaly`'s extra columns, which would probably require a little hacking in alerts-table to make sure that extra columns will not be populated by accident if they are not present in a given alert.
